### PR TITLE
add Nguyen_HanhNong bot to the SOR lab

### PR DESCRIPTION
**Pull request by Nguyen-HanhNong**
pull request to setup Nguyen_HanhNong bot to the Summer of Rust lab

### CODE REVIEW

**Comment by AngelOnFira** [comment]
Looks like CI is failing, you'll need to run `cargo fmt` and `cargo clippy` :) Also, I think you're going to need to change your struct name by removing the underscore. That should be fine I think :100:

File: src/bots/mod.rs [hunk 1]
```diff
@@ -50,6 +52,7 @@ impl Bots {
             Bots::AlwaysBetray => Box::new(AlwaysBetray::new()),
             Bots::GrimTrigger => Box::new(GrimTrigger::new()),
             Bots::Detective => Box::new(Detective::new()),
+            Bots::NguyenHanhNong => Box::new(nguyen_hanh_nong::NguyenHanhNong::new()),
```
Comment (AngelOnFira) [inline src/bots/mod.rs:55]: Can we change this to a use statement? At the top of the page, there are some examples of how this is done with the other bots :)

**Review by AngelOnFira — CHANGES_REQUESTED** [review]
Just one small thing then this will be good to go :)

**Comment by Nguyen-HanhNong** [comment]
accidentally deleted your comment, my bad. It kept appearing in vscode every time I tried to change the declaration at line 56 .

**Review by AngelOnFira — APPROVED** [review]

**Pull request merged**